README.md

```diff
@@ -202,7 +202,50 @@ npm install tslint-sonarts -g           # or install globally
 ```
 tslint --project ./tsconfig.json 'src/**/*.{ts,tsx}'
 ```
-_[We also have a plugin for ESLint](https://github.com/SonarSource/eslint-plugin-sonarjs)_
+
+## Use in ESLint
+
+It is possible to use TSLint plugin inside ESLint. To do that follow these steps:
+* Install packages `tslint`, `eslint`, `tslint-sonarts` (our plugin), `@typescript-eslint/parser` (parser of TypeScript for ESLint) and `@typescript-eslint/eslint-plugin-tslint` (runner of TSLint rules inside ESLint).
+* Create ESLint configuration file
```

Comment:
I think we could add here a command like
```
npm -i tslint eslint tslint-sonarts @typescript-eslint/parser @typescript-eslint/eslint-plugin-tslint --save-dev
```
so you can just copy&paste in the terminal